packages/console/src/pages/Applications/index.tsx

```diff
@@ -19,23 +28,35 @@ const Applications = () => {
       <table className={styles.table}>
         <thead>
           <tr>
-            <td className={styles.applicationName}>Application Name</td>
-            <td>Client ID</td>
+            <td className={styles.applicationName}>{t('applications.application_name')}</td>
+            <td>{t('applications.application_name')}</td>
           </tr>
         </thead>
         <tbody>
-          <tr>
-            <td>
-              <ItemPreview
-                title="Default App"
-                subtitle="Single Page Application"
-                icon={<ImagePlaceholder />}
-              />
-            </td>
-            <td>
-              <CopyToClipboard value="RUMatENw0rFWO5aGbMI8tY2Qol50eOg3" />
-            </td>
-          </tr>
+          {error && (
+            <tr>
+              <td colSpan={2}>error occurred: {error.metadata.code}</td>
+            </tr>
+          )}
+          {isLoading && (
+            <tr>
+              <td colSpan={2}>loading</td>
+            </tr>
+          )}
+          {data?.map(({ id, name, type }) => (
+            <tr key={id}>
+              <td>
+                <ItemPreview
+                  title={name}
+                  subtitle={t(applicationTypeI18nKey[type])}
+                  icon={<ImagePlaceholder />}
+                />
+              </td>
+              <td>
+                <CopyToClipboard value={id} />
```

Comment:
Should we provide a fixed width to all the client id fields? Currently the width varies from id to id

<img width="197" alt="image" src="https://user-images.githubusercontent.com/12833674/156730561-6a791755-a247-4af8-b02a-e10271c3bf0f.png">

Reply:
discussed offline